stdlib/ballerina-http/src/main/ballerina/net.http/http_connection.bal

```diff
@@ -5,6 +5,9 @@ public const string CONTENT_LENGTH = "content-length";
 
 const string HEADER_KEY_LOCATION = "Location";
 
+documentation {
+    Represents HTTP connection.
```

Comment:
Shall we elaborate on this? i.e: how and where this `Connection` struct fits in our endpoint stuff

Reply:
Converting BStructs to structs for each and every request can introduce performance bottlenecks.